atom/browser/atom_resource_dispatcher_host_delegate.cc

```diff
@@ -86,9 +86,10 @@ void OnPdfResourceIntercepted(
   // The URL passes the original pdf resource url, that will be requested
   // by the webui page.
   // chrome://pdf-viewer/index.html?src=https://somepage/123.pdf
+  GURL escaped_url(net::EscapeUrlEncodedData(original_url.spec(), true));
```

Comment:
* Result of `net::EscapeUrlEncodedData` can be used directly in the final url creation below, `escaped_url` `GURL` is not required.

* the `use_plus` parameter should be invoked with false, since we use `net::UnescapeRule::SPACES ` below.